src/router.ts

```diff
@@ -52,22 +46,22 @@ const routes:RouteRecordRaw[] = [
   },
   {
     path: '/',
-    component: Default,
+    component: () => import('@/layouts/default/Default.vue'),
     children: [
       {
         beforeEnter: (to: any, from: any, next: any) => {
           to.params.m = to.redirectedFrom
           next()
         },
         path: '/404',
-        component: NotFound,
+        component: () => import('@/views/NotFound.vue'),
         meta: {
           title: 'Page not found'
         }
       },
       {
         path: '/error',
-        component: Error,
+        component: () => import('@/views/Error.vue'),
```

Comment:
I brought back lazy-loading in `router.ts`; recent PRs did not fix mime-type problem. After this is merged I will flip dev CodePipeline to use my own working branch to squash that bug. 🪰